shell/browser/api/electron_api_app.cc

```diff
@@ -711,6 +711,12 @@ void App::OnFinishLaunching(base::Value::Dict launch_info) {
   // applications. Only affects pulseaudio currently.
   media::AudioManager::SetGlobalAppName(Browser::Get()->GetName());
 #endif
+  // SAP-14667 handling notification from previous session requires
```

Comment:
All these `SAP` comments need to be removed.  The comment body could stay depending on the context, but the `SAP` numbers need to go.

Reply:
Please consider commit https://github.com/electron/electron/commit/88f173b1bf6a883f0de2c06c1eeb74942a641bb0